src/main/webapp/assets/bootstrap/css/bootstrap-theme.css

```diff
@@ -0,0 +1,470 @@
+/*!
```

Comment:
Bootstrap is already included in our assets folder. Please re-map all your Bootstrap references to assets/venders/bootstrap-3.3.7-dist